pkg/container/qemu_runner.go

```diff
@@ -1188,3 +1115,187 @@ func randomPortN() (int, error) {
 
 	return l.Addr().(*net.TCPAddr).Port, nil
 }
+
+func generateCpio(ctx context.Context) (string, error) {
+	/*
+	 * we only build once, useful for local development, we
+	 * cahce it into kernel/ARCH/melange-guest.initramfs.cpio
+	 * if present, we nop and return, else we build it.
```

Comment:
Perhaps we should use this, so it doesn't dirty trees and potentially shares across repos?
https://pkg.go.dev/os#UserCacheDir

This should have some sort of expiration, I'd think...

Reply:
Right, done :+1: